main.go

```diff
@@ -1070,11 +1070,20 @@ func viewTrainingRuns(c *gin.Context) {
 
 	rows := []gin.H{}
 	for _, training_run := range training_runs {
+		var bestNetwork string
```

Comment:
I think this could potentially done with a preload on the Find call above. But this should be fine for now.

Reply:
I'll give it a try

Reply:
Done